src/ios/CordovaCall.m

```diff
@@ -518,18 +519,18 @@ - (void)provider:(CXProvider *)provider didDeactivateAudioSession:(AVAudioSessio
 
 - (void)provider:(CXProvider *)provider performAnswerCallAction:(CXAnswerCallAction *)action
 {
-    [self setupAudioSession];
+    // [self setupAudioSession];
```

Comment:
It seems Apples wants `AudioSession` before `fulfill`

from https://developer.apple.com/documentation/callkit/making-and-receiving-voip-calls

> After the recipient answers the call, the system calls the provider delegate’s [provider(_:perform:)](https://developer.apple.com/documentation/callkit/cxproviderdelegate/provider(_:perform:)-2lem5) method. In your implementation of that method, configure an [AVAudioSession](https://developer.apple.com/documentation/AVFAudio/AVAudioSession) and call the [fulfill()](https://developer.apple.com/documentation/callkit/cxaction/fulfill()) method on the action object when it finishes, as in the code example below:

Reply:
in case of (our use case where) media is through the JS context (app), then maybe we shouldn't `fulfill` but do a round-trip to the app, then the app will let the native side know it's done.

Reply:
I think this might actually be fine to keep in, it's not creating a new `AVAudioSession` but rather grabbing the shared instance across the entire app.

`cordova-plugin-ioswebrtc` also does the same, https://github.com/cordova-rtc/cordova-plugin-iosrtc/blob/bd2a7386b6b79a7d74aa0da69c6c942807eb98c8/src/PluginRTCAudioController.swift#L43

Reply:
I didn't look too deeply into the `AVAudioSession` stuff as I was focusing on other things...